.gitconfig

```diff
@@ -0,0 +1,18 @@
+# Include Grml's gitconfig
+[include]
+	path = ~/.grml-etc-core/etc/skel/.gitconfig
+
+# Those are #!'s customizations
+[push]
+        default = simple
+[hub]
+        protocol = https
```

Comment:
No idea why it isn't the default (ask Github?), but it is useful because otherwise the bare git protocol is used (which doesn't ensure integrity of the transfers).